src/components/molecules/all-movies-table/columns.tsx

```diff
@@ -0,0 +1,171 @@
+"use client";
+
+import { ColumnDef, RowData } from "@tanstack/react-table";
+import {
+  CreditDbI,
+  TmdbGenreI,
+  TmdbProductionCountryI,
+  FetchMoviesReturnType,
+} from "@/lib/definitions";
+import { calculateMovieInterest, formatListItemsWithDelimiter } from "./utils";
+import { useSession } from "next-auth/react";
+import MultipleItemsCellWrapper from "./multiple-items-cell-wrapper";
+import { ReactionRate } from "@/components/molecules/movie-reaction-panel";
+import TableHeader from "./table-header";
+import { Progress } from "@/components/ui/progress";
+import type { MovieReaction, User } from "@prisma/client";
+
+declare module "@tanstack/react-table" {
+  interface TableMeta<TData extends RowData> {
+    usersCount: number;
+  }
+}
+
+export const columns: ColumnDef<FetchMoviesReturnType["movies"][number]>[] = [
+  {
+    accessorKey: "title",
+    header: () => <TableHeader text={"Title"} />,
+    cell: ({ row }) => {
+      const imdbId = row.getValue("imdb_id") as string;
```

Comment:
a jakby użyć `row.original.imdb_id`? chyba że to gorsza praktyka ale nie wiem

Reply:
mysle, ze lepiej sie trzymac tego getValue, bo to jest zalecane podejscie. https://tanstack.com/table/v8/docs/guide/rows
<img width="871" alt="Screenshot 2024-12-15 at 13 19 06" src="https://github.com/user-attachments/assets/fb05a279-0f12-466f-bbd6-8f5e75d09892" />